packages/node/src/integrations/linkederrors.ts

```diff
@@ -47,7 +47,7 @@ export class LinkedErrors implements Integration {
       const hub = getCurrentHub();
       const self = hub.getIntegration(LinkedErrors);
       const client = hub.getClient<NodeClient>();
-      if (client && self) {
+      if (client && self && self._handler && typeof self._handler === 'function') {
         await self._handler(client.getOptions().stackParser, event, hint);
```

Comment:
I don't understand how this happens either but this should fix it for now. Will try to investigate this more on Monday. Since we're cutting a patch on Monday, I'll leave this open till then

Reply:
If I had to guess it's due to multiple `@sentry/node`s being installed - maybe through transitive deps?

Reply:
I agree that a version conflict might cause this. Apart from that I don't see how this method would not exist while the class instance obviously does exist.

Reply:
Merging this for now as I think the check doesn't hurt us

Reply:
I checked out package-lock.json when we had 7.8.0 installed and can't see any references to other versios of sentry/node in there.